bindgen-tests/tests/expectations/tests/strings_cstr.rs

```diff
@@ -0,0 +1,22 @@
+#![allow(
+    dead_code,
+    non_snake_case,
+    non_camel_case_types,
+    non_upper_case_globals
+)]
+
+#[allow(unsafe_code)]
+pub const MY_STRING_UTF8: &::std::ffi::CStr = unsafe {
+    const BYTES: &[u8; 14] = b"Hello, world!\0";
```

Comment:
is it possible to omit this intermediate constant and do `CStr::from_bytes_with_nul_unchecked(b"Hello, world!\0")` instead?

Reply:
I think it's possible in this case.

I used this pattern in `cmacro` since it will add `.as_ptr()` inside function macros, and e.g. `b"Hello, world!\0".as_ptr()` would be invalid. In the `CStr` case that doesn't happen though.